src/test/java/com/cloudant/tests/util/Utils.java

```diff
@@ -33,4 +44,58 @@ public static String getHostName(String account) {
         }
 
     }
+
+    public static void removeReplicatorTestDoc(CloudantClient account, Response replicatorResponse)
+            throws Exception {
+
+        //Find replicator document for removal
+        ReplicatorDocument replicatorDoc = waitForReplicatorToComplete(account, replicatorResponse);
+
+        Response removeResponse = account.replicator()
+                .replicatorDocId(replicatorDoc.getId())
+                .replicatorDocRev(replicatorDoc.getRevision())
+                .remove();
+
+        assertThat(removeResponse.getError(), is(nullValue()));
+    }
+
+    public static ReplicatorDocument waitForReplicatorToComplete(CloudantClient account, Response
+            replicatorResponse)
+            throws Exception {
+        ReplicatorDocument replicatorDoc = null;
+
+        boolean finished = false;
+
+        long startTime = System.currentTimeMillis();
+        long timeout = startTime + TIMEOUT_CONSTANT;
+        //initial wait of 100 ms
+        long delay = 100;
+
+        while (!finished && System.currentTimeMillis() < timeout) {
+            //Sleep before finding replication document
+            Thread.sleep(delay);
+
+            replicatorDoc = account.replicator()
+                    .replicatorDocId(replicatorResponse.getId())
+                    .find();
+
+            //If replicator state is completed or triggered, return true
+            //If false, continue in the while loop until replicator is complete
+            if (replicatorDoc != null && replicatorDoc.getReplicationState() != null
+                    && (replicatorDoc.getReplicationState().equalsIgnoreCase("completed")
+                    || (replicatorDoc.getContinuous() != null && replicatorDoc.getContinuous()
+                    && replicatorDoc.getReplicationState().equalsIgnoreCase("triggered")))) {
```

Comment:
The method name isn't accurate given it only waits for continuous replications to start.

I'd split this up into `waitForReplicationToStart` and `waitForReplicationToComplete` so you don't have this nasty-to-understand if statement and incorrect method naming.